DEV-ADR.md

```diff
@@ -2,6 +2,50 @@
 
 ---
 
+<!--
+Template of ADR
+
+## ID. TITLE
+
+date: 2025-XX-XX
+status: Accepted
+
+### Context
+
+To complete
+
+### Decision
+
+To complete
+
+### Consequences
+
+To complete
+-->
+
+## 3. Specific Mithril status code
+
+date: 2025-03-21
+status: Accepted
+
+### Context
+
+In exchanges between the signer and the aggregator, we need to retrieve the reason why a request was unsuccessful.
+Error handling will depend on the specific functional case of Mithril.
+We could have reused existing HTTP codes, but they are too general and could be returned for cases other than the one we wish to isolate.
+
+### Decision
+
+We therefore decided to return specific server error codes starting at 550.
+
+Status codes introduced:
+
+- 550: registration round not yet opened
```

Comment:
Maybe we want to specify only the ranges we allocate for errors `4XX` and `5XX` and not list them all here as they will be explained in the OpenAPI?